src/syft/core/pointer/pointer.py

```diff
@@ -243,7 +262,8 @@ def request(
         request_name: str = "",
         name: str = "",
         reason: str = "",
-    ) -> None:
+        block: bool = False,
```

Comment:
I do not think we need this here, we need only for get. So I think we can remove this?

Reply:
we still need to make the internal call from .get() to get the status, if the call is accepted or rejected